spec/workflow/states/map_spec.rb

```diff
@@ -26,7 +26,7 @@
         "Type"           => "Map",
         "InputPath"      => "$.detail",
         "ItemsPath"      => "$.shipped",
-        "MaxConcurrency" => 0,
```

Comment:
I was wondering why my spec was hanging :laughing: 
Add a check to raise a workflow error if MaxConcurrency was 0